Makefile

```diff
@@ -1,9 +1,18 @@
-.PHONY: vendor
-vendor:
-	docker build -t ebpf-exporter-build .
-	docker run --rm -v $(CURDIR):/go/ebpf_exporter --workdir /go/ebpf_exporter --entrypoint /bin/bash ebpf-exporter-build -c "go get -u ./... && go mod vendor && go mod tidy"
+ifeq ($(VERSION),)
+	VERSION = latest
+endif
+
+IMAGE = ebpf-exporter:$(VERSION)
+
+container:
+	docker build -t $(IMAGE) .
+.PHONY: container
+
+push: container
+	docker push $(IMAGE)
```

Comment:
I do not intend to push the image anywhere, so let's not add things that will not be used.